File: ABC/ABC112/B/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  int N, T;
  cin >> N >> T;

  int answer = 0;

  for (int i = 0; i < N; i++) {
    int c, t;
    cin >> c >> t;
    if (t <= T) {
      if (answer == 0){
        answer = c;
      } else {
        answer = min(answer, c);
      }
    }
  }

  if (answer == 0) {
    cout << "TLE" << endl;
  } else {
    cout << answer << endl;
  }
  
}

File: ABC/ABC112/B/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    
    int N = sc.nextInt();
    int T = sc.nextInt();

    int[] c = new int[N];
    int[] t = new int[N];

    for (int i = 0; i < N; i++) {
      c[i] = sc.nextInt();
      t[i] = sc.nextInt();
    }

    sc.close();

    int min = 1001;

    for (int i = 0; i < N; i++) {

      if (t[i] <= T) {
        min = Math.min(min, c[i]);
      }

    }

    if (min == 1001) {
      System.out.println("TLE");
    } else {
      System.out.println(min);
    }

  }

}